### ISSUE TITLE
APDS / MicroOLED / WiFi missing in "libraries"

### ISSUE THREAD

**Issue by chatelao** [sparkfun/PSoC_Arduino_Support#11]
Compiling the "Uno_blocks" in "PSoC Creator 3.3" reports a problem due to missing files for the libraries.

**Comment by chatelao** [comment]
Is there anywhere a step-by-step description, how to get the project compiling after fetching from github?

**Comment by mhord** [comment]
No; the [hookup guide](https://learn.sparkfun.com/tutorials/freesoc2-introduction?_ga=1.93428763.725448541.1330116044#getting-started-with-psoc-creator) does include a getting started walkthrough of Creator (it's a bit dated, now, in 2.2, but most things haven't changed much).

This is *not* a trivial project to work with. Creator does not, by default, support C++, so to get the Arduino core support ported over I had to manually adjust the switches being passed to the compiler, among other things.

I've just updated the project to fix a few issues relating to Creator 3.3 (mostly due to the new version of GCC bundled with it); it should build without errors now.

**Comment by chatelao** [comment]
Thanks a lot, we are not in for the easy side (yet). It is such a cool MCU.
